docs/basics/part-8b-using-earthly-ci.md

```diff
@@ -0,0 +1,71 @@
+The code of this example is available in the [earthly/earthly-example-ci repository on GitHub](https://github.com/earthly/earthly-example-ci).
```

Comment:
Italicize this. "_The code of this example is available in the [earthly/earthly-example-ci repository on GitHub](https://github.com/earthly/earthly-example-ci)._"